modules/manage/pages/security/authorization/rbac.adoc

```diff
@@ -0,0 +1,131 @@
+
+Using RBAC, you can define roles to reflect organizational structure or job duties to gives a coherent approach to access management. This approach decouples users and permissions through the assignment of roles. This allows a one to many mapping of a given role to many users, thereby dramatically reducing the number of custom policies needed for a resource from one per user to one per group of users.
+
+image::shared:rbac-overview.png[RBAC multiplicity]
+
+=== RBAC terminology
+
+Under the RBAC framework, you create *roles*, grant *permissions* to those roles, and assign the roles to *users*. When you change the permissions for a given role, all principals with that role automatically gain the modified permissions. You grant or deny permissions for a role by creating an ACL and specifying the RBAC role as the allowed or denied principal respectively.
+
+==== Roles
+
+A role is a named collection of ACLs which can be assigned to zero or more users. Redpanda provides two default roles, `user` and `admin`:
+
+* `user` - All authorized users have this role by default. There are no ACLs assigned to this role until you add them. This means users have no permissions to interact with any resources you add to your Redpanda cluster until you add relevant permissions to this or another role.
+* `admin` - By default, no users have this role. The default admin role provides full access to all resources in your Redpanda installation.
+
+==== Policy conflicts
+
+You can assign a combination of ACLs and roles to any given principal. ACLs can contain allow permissions, deny permissions, or a combination of both. As a result, users may at times have conflicting policies. If this situation arises, the user is permitted to perform an operation if and only if:
+
+* There does not exist a DENY permission matching the operation.
+* There exists an ALLOW permission matching the operation.
+
+== Managing users and roles
+
+Redpanda allows you to manage your RBAC configurations through both RPK and the web console. Examples of each are given here.
+
+=== Create a role
+
+Creating a new role is a two-step process. First you define the role, giving it a unique and descriptive name. Second, assign that role as the allowed or denied principal to one or more ACLs. This defines the permissions associated with the role.
+
+Using RPK, administrators create a new role by issuing the command:
+
+[,bash]
+----
+rpk role create [ROLE_NAME]
+----
+
+Once the role is created, administrators assign the role as the allowed principal for one or more ACLs. You can modify existing ACLs or create new ones. For example:
+
+[,bash]
+----
+rpk acl create ... --allow-principal Role:[ROLE_NAME]
```

Comment:
This is a bit stale 

```suggestion
rpk acl create ... --allow-role [ROLE_NAME]
```
Please lmk if there are docs hanging around that still contain this usage. Keeping them all up to date is...challenging 😅